backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/IBPartnerBL.java

```diff
@@ -219,6 +219,9 @@ public enum IfNotFound
 		@NonNull
 		Comparator<User> comparator = Comparator.comparing(User::getName);
 
+		@Default
+		boolean onlyMailEnabled = false;
```

Comment:
i think the field should rather be named `onlyIfInvoiceEmailEnabled`

Reply:
done